app/models/user.rb

```diff
@@ -6,7 +6,7 @@ class User < ApplicationRecord
   AVAILABLE_BAY_BACKGROUND_COLORS = %w[modele gestion cluster].freeze
   AVAILABLE_BAY_ORIENTATIONS = %w[front back].freeze
 
-  enum :role, %i[user vip admin]
+  enum :role, { :user => 0, :vip => 1, :admin => 2 }
```

Comment:
Don't we prefer this?

```suggestion
  enum :role, { user: 0, vip: 1, admin: 2 }
```